src/ucs/sys/topo.h

```diff
@@ -75,17 +75,31 @@ ucs_status_t ucs_topo_get_distance(ucs_sys_device_t device1,
                                    ucs_sys_dev_distance_t *distance);
 
 
+/**
+ * Return system device name in BFD format: <domain>:<bus>:<device>.<function>
+ *
+ * @param [in]  sys_dev  System device id, as returned from
+ *                       @ref ucs_topo_find_device_by_bus_id
+ * @param [out] buffer   String buffer, filled the device name
+ * @param [in]  max      Maximal size of @a buffer
```

Comment:
what does `@a` mean?
should it be `@buffer`?

Reply:
`@a` in doxygen refers to a parameter (in this case, `buffer`)

Reply:
i see
thanks